js/classes/entities/midground/enemies/enemyBase.js

```diff
@@ -177,7 +178,7 @@ class EnemyBase {
             // if we're pursuing Chad, update the target position to Chad's position
             // also avoid changing direction in the middle of an attack animation
             if (this.enemy.state === "pursue" && this.enemy.action != "attacking") {
-                this.setTargetX(CHAD.pos.x);
+                this.setTargetX(CHAD.pos.x + CHAD.scaleBoundingBoxOffset().x);
```

Comment:
Good catch! This should make it so the enemies attack the right portion of Chad.

Reply:
Not quite, it just accounts for the empty space between the left edge of his spritesheet and the left edge of his body.